ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline.yml

```diff
@@ -226,19 +226,6 @@ spec:
           workspace: pipeline
           subPath: src
 
-    - name: get-organization
```

Comment:
if we get rid of this task from the pipeline, are not going to be issues with subsequent tasks that are using it's value? for example `reserve-operator-name`